CHANGELOG.md

```diff
@@ -35,6 +34,11 @@ Initial release prepared by [@chshersh](https://github.com/chshersh).
   Add `MonadUnliftIO` instance for the `CliApp` monad
   (by [@charrsky](https://github.com/charrsky))
 
+### Changed
+
+- [#90](https://github.com/chshersh/iris/issues/90):
+  Use Data.Text instead of ByteString for coloured output
```

Comment:
Thanks for adding an entry!

Could you move this inside the `## [Unreleased]` section above in the changelog?